src/components/Tag.tsx

```diff
@@ -0,0 +1,43 @@
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import React, { FunctionComponent, ReactNode, ReactElement } from "react";
+
+import "govuk-frontend/govuk/components/tag/_tag.scss";
+
+/**
+ * The property types for {@link Tag}.
+ *
+ * @property {string} [id]
+ * @property {string} [className]
+ * @property {ReactNode} children - The tag text, for example, 'Alpha'
+ */
+export interface TagProps {
+  id?: string;
+  className?: string;
+  children: ReactNode;
+}
+
+/**
+ * A component for indicating the state of a process or product.
```

Comment:
```suggestion
 * A component for indicating the status of something, such as an item on a task list or a phase banner.
```

The design system's explanation is good. Let's reuse it if we can.